crates/aggchain-proof-contracts/src/lib.rs

```diff
@@ -221,49 +226,49 @@ where
         // Static call on the new LER
         host_execute(
             bridge_address,
-            &mut executor,
+            &mut sketch,
             BridgeL2SovereignChain::getRootCall {},
-            StaticCallStage::NewHashChain(HashChainType::InsertedGER),
+            StaticCallStage::NewLer,
```

Comment:
This change is intended?

Reply:
yes that's a fix, it wasn't set to the proper value